es-ds-docs/pages/molecules/file-input.vue

```diff
@@ -0,0 +1,311 @@
+<template>
+    <div>
+        <h1>
+            File input
+        </h1>
+
+        <div class="mb-500">
+            <es-file-input
+                :upload-urls="uploadUrls"
+                :max-file-size-mb="10"
+                :file-types="['image/png', 'application/pdf']"
+                @fileSizeError="fileSizeError"
+                @fileTypeError="fileTypeError"
+                @fileIsAFolderError="fileIsAFolderError"
+                @readyToUpload="readyToUpload"
+                @uploadSuccess="uploadSuccess"
+                @uploadFailure="uploadFailure"
+                @fileDataRead="fileDataRead">
+                <template #cta>
+                    <h2 class="d-none d-md-inline-block font-size-500 text-center my-200">
+                        Drag and drop your files or
+                    </h2>
+                    <p class="d-inline-block d-md-none my-200">
+                        <b>Don't forget:</b> Make sure to upload an image of both sides of your bill.
+                    </p>
+                </template>
+                <template #helpText>
+                    <p class="d-none d-md-inline-block mb-0 mt-200">
+                        Please upload your file as a PDF or JPG.
```

Comment:
Another pre-existing issue: the mime types are actually restricted to PDF and PNG, so maybe we should update these notes

Reply:
Good catch!

Reply:
Good catch!